imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -123,6 +123,110 @@ def get_spacecraft_histogram(
     return hist, latitude, longitude, n_pix
 
 
+def get_helio_histogram(
+    time: NDArray,
+    vhat: NDArray,
+    energy: NDArray,
+    energy_bin_edges: list[tuple[float, float]],
+    nside: int = 128,
+    nested: bool = False,
+) -> tuple[NDArray, NDArray, NDArray, NDArray]:
+    """
+    Compute a 3D histogram of the particle data using HEALPix binning.
+
+    Parameters
+    ----------
+    time : np.ndarray
+        Median time of pointing in et.
+    vhat : tuple[np.ndarray, np.ndarray, np.ndarray]
+        The x,y,z-components of the unit velocity vector.
```

Comment:
```suggestion
        The x,y,z-components of the unit velocity vector (spacecraft frame).
```
I think the above is more accurate, right?